client/my-sites/plan-features-comparison/style.scss

```diff
@@ -1,13 +1,15 @@
 $plan-features-header-banner-height: 20px;
 $plan-features-sidebar-width: 272px;
 
-.plan-pill.is-in-signup {
-	right: 52%;
-	transform: translate(50%);
-	border-radius: 4px;
-	background-color: var(--studio-green-5);
-	color: var(--studio-gray-80);
-	font-size: $font-body-small;
+.plan-features-comparison__content {
```

Comment:
`is-in-signup` isn't applied by the plugins plans page. This shouldn't be necessary.

@candy02058912 pointed this out in slack:

https://github.com/Automattic/wp-calypso/blob/f67056fc062f8fafc7b3009f6bbc06ff3f8d9369/client/my-sites/plan-features-comparison/header.jsx#L45

Can we fix this instead?

Reply:
Will repurpose this PR to fix this and for our own implementation, we can continue with what @lupus2k did [here](https://github.com/Automattic/wp-calypso/pull/68123/files#diff-fb7f88f187f934c76e2271680a8405344da50fc8cb2ab6aac509459a02acb6cfR880-R896).